android/Staccato_AN/app/src/main/java/com/on/staccato/presentation/momentcreation/MomentCreationActivity.kt

```diff
@@ -99,12 +99,20 @@ class MomentCreationActivity :
 
     override fun onResume() {
         super.onResume()
-        if (viewModel.isPlaceSearchClicked.getValue() == true) {
-            viewModel.setIsPlaceSearchClicked(false)
+        if (viewModel.isPlaceSearchClicked.value != true) {
             checkLocationSetting()
         }
     }
 
+    override fun onPause() {
+        super.onPause()
+        if (autocompleteFragment.isVisible) {
+            viewModel.setIsPlaceSearchClicked(true)
+        } else {
+            viewModel.setIsPlaceSearchClicked(false)
+        }
```

Comment:
이렇게 변경한다면 더 깔끔하면서도 코드의 동작을 명시적으로 나타낼 수 있을 것 같아요!

```suggestion
        val hasPlaceSearchClicked = autocompleteFragment.isVisible
        viewModel.setIsPlaceSearchClicked(hasPlaceSearchClicked)
```

Reply:
앗 이 부분을 놓쳤네요🤣 감사합니다 호두 반영 완료했습니다!!